web/src/app/browser/src/hardwareEventKeyboard.ts

```diff
@@ -257,7 +257,7 @@ export default class HardwareEventKeyboard extends HardKeyboard {
   }
 
   get activeKeyboard(): Keyboard {
-    return this.contextManager.activeKeyboard.keyboard;
+    return this.contextManager.activeKeyboard?.keyboard;
```

Comment:
I see `keyPress` calls this function:

https://github.com/keymanapp/keyman/blob/a404c20ddf12ffbe8dcd36bd51ad6343ec37cd9d/web/src/app/browser/src/hardwareEventKeyboard.ts#L409

It may be that it never gets there because of other preconditions but it should probably also be fixed.

The name of the function is a bit confusing -- in some places `activeKeyboard` refers to a type `{keyboard: Keyboard, metadata: KeyboardStub}` from contextManager.ts, and in other places, it refers to a `Keyboard` type. Something for another day though.